RELEASE-NOTES.md

```diff
@@ -9,669 +12,344 @@
-  provide more flexibility in operational
-  situations. [riak_cs/#1021](https://github.com/basho/riak_cs/pull/1021)
+* [Riak 1.4 series release notes](https://github.com/basho/riak/blob/1.4/RELEASE-NOTES.md)
+* [Riak 2.0 series release notes](https://github.com/basho/riak/blob/2.0/RELEASE-NOTES.md)
+* [Riak CS 1.4 series release notes](https://github.com/basho/riak_cs/blob/develop/RELEASE-NOTES.md#riak-cs-145-release-notes)
+* [Riak CS 1.5 series release notes](https://github.com/basho/riak_cs/blob/develop/RELEASE-NOTES.md#riak-cs-154-release-notes)
+* Riak CS 2.0 release notes here
 
-## Bugs Fixed
+## Upgrading From Riak CS < 1.4
 
-- Fix storage usage calculation bug where data for deleted buckets
-  would be included in the calculation
-  results. [riak_cs/#996](https://github.com/basho/riak_cs/pull/996)
+Basho only support 2 older versions from the latest release. For Riak
+CS 2.0 case, only upgrades from 1.4 series and 1.5 series are
+supported. Basically this pattern is out of the scope of this
+document.
 
-# Riak CS 1.5.2 Release Notes
+To upgrade to 2.0 from these versions, operators need another extra
+step to upgrade the whole system to Riak CS 1.4.5 or Riak
+CS 1.5.4. Upgrading to 1.5.4 would rather be recommented than
+to 1.4.5. Thus underlying Riak would also better upgraded to 1.4
+series - hopefully the latest 1.4.12.
 
-## Additions
+Notes TBD: any caveats on upgrading to CS 1.4/1.5? or references.
 
-- Improved logging around connection failures with Riak
-  [riak_cs/#987](https://github.com/basho/riak_cs/pull/987).
-- Add amendment log output when storing access stats into Riak failed
-  [riak_cs/#988](https://github.com/basho/riak_cs/pull/988). This
-  prevents losing access stats logs in cases of temporary connection
-  failure between Riak and Riak CS. Access logs are stored in
-  `console.log` at the `warning` level.
-- Add script to repair invalid garbage collection manifests
-  [riak_cs/#983](https://github.com/basho/riak_cs/pull/983). There is
-  a [known issue](https://github.com/basho/riak_cs/issues/827) where
-  an active manifest would be stored in the GC bucket. This script
-  changes invalid state to valid state.
+# General Instructions
 
-## Bugs Fixed
+## Riak CS and Riak are in the same box
 
-- Fix Protocol Buffer connection pool (`pbc_pool_master`) leak
-  [riak_cs/#986](https://github.com/basho/riak_cs/pull/986) . Requests
-  for non-existent buckets without an authorization header and
-  requests asking for listing users make connections leak from the
-  pool, causing the pool to eventually go empty. This bug was introduced
-  in release 1.5.0.
+General Instructions are to repeat this overview on every node:
 
-# Riak CS 1.5.1 Release Notes
+1. Stop Riak CS process
+2. Stop Riak process
+3. Backup all configuration files
+4. Upgrade Riak CS software
+5. Upgrade Riak software
+6. Update Riak configuration
+7. Migrate Riak CS configuration
+8. Start Riak
+9. Start Riak CS
```

Comment:
In "Riak CS nodes not corriding in a same box with live Riak node" section, upgrading steps include uninstall and install packages. This steps say upgrade.